redis/src/client.rs

```diff
@@ -190,6 +232,59 @@ impl Client {
         }
     }
 
+    /// Returns an async connection from the client.
+    #[cfg(any(feature = "tokio-comp", feature = "async-std-comp"))]
+    #[cfg_attr(
+        docsrs,
+        doc(cfg(any(feature = "tokio-comp", feature = "async-std-comp")))
+    )]
+    pub async fn get_multiplexed_async_connection_with_config(
```

Comment:
`get_multiplexed_async_connection_with_timeouts` should call this

Reply:
same here https://github.com/redis-rs/redis-rs/blob/3cf579fd0da327d018559f1981e6cc27d6fe9cb3/redis/src/aio/connection_manager.rs#L195C20-L195C66

Reply:
Done. Good thought